plugin/whoami/README.md

```diff
@@ -23,6 +23,11 @@ If CoreDNS can't find a Corefile on startup this is the _default_ plugin that ge
 it can be used to check that CoreDNS is responding to queries. Other than that this plugin is of
 limited use in production.
 
+[Read the blog post][blog] on how this plugin is built, or [explore the source code][code].
```

Comment:
Can you add an "Also See" Section at the end and move this there?

(more inline with the other plugin's README)